src/Kafka/KafkaInstances/KafkaInstances.stories.tsx

```diff
@@ -16,11 +12,11 @@ export default {
   args: {
     getUrlForInstance: (row) => `/${row.id}`,
     isRowSelected: () => false,
-    canChangeOwner: (row) => row.owner === "baz-owner",
+    canChangeOwner: (row) =>
+      row.owner === "baz-owner" || row.status === "suspended",
```

Comment:
I know it's a story, but can suspended instances be moved to another owner?

Reply:
changed the story. actually, I am writing a test to check the actions items based on the status but for some reason not able to get the action button based on aria-label,

Reply:
I am writing a test to check the actions items based on status, but unable to access the action button, and the test is failing with the below error https://github.com/redhat-developer/app-services-ui-components/actions/runs/3581280261/jobs/6024192336#step:4:484. @riccardo-forina do you have any idea about this

Reply:
I'm not super sure what it could be, main builds fine 😕

Reply:
I get this warning when I tried to access the action button in the instance row, I can see the same waring message locally and in stage